universe/Tech.cpp

```diff
@@ -376,8 +373,7 @@ TechManager::TechManager() {
 #ifdef OUTPUT_TECH_LIST
     for (const Tech* tech : m_techs.get<NameIndex>()) {
         std::cerr << UserString(tech->Name()) << " (" 
-                  << UserString(tech->Category()) << " "
-                  << UserString(boost::lexical_cast<std::string>(tech->Type())) << ") - "
```

Comment:
@geoffthemedio you removed the tech type (theory vs application) two year ago (or so).